tortoise/backends/sqlite/schema_generator.py

```diff
@@ -19,3 +21,21 @@ def _table_comment_generator(self, table: str, comment: str) -> str:
 
     def _column_comment_generator(self, table: str, column: str, comment: str) -> str:
         return f" /* {self._escape_comment(comment)} */"
+
+    def _column_default_generator(
+        self,
+        table: str,
+        column: str,
+        default: Any,
+        auto_now_add: bool = False,
+        auto_now: bool = False,
+    ) -> str:
+        default_str = " DEFAULT"
+        if not (auto_now or auto_now_add):
```

Comment:
I think `auto_now` implies `auto_now_add`?

Reply:
Yep: https://github.com/tortoise/tortoise-orm/blob/develop/tortoise/fields/data.py#L295
You can simplify this?

Reply:
Yeah, that's right.